.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: 🧪 Test
```

Comment:
Can you name it something like: `Test PR from fork`
I rather we are more explicit so its clear for new comers what this is about

Reply:
yes, that seems good.